modules/commandModule/commandWorker_flight.py

```diff
@@ -15,4 +19,34 @@ def flight_command_worker(pipelineIn, pipelineOut):
 
 	#TODO: Write logic to get POGI
 
-	
\ No newline at end of file
+
+def pogi_command_worker(pause, exitRequest, pipelineOut):
+    """
+    Producer worker function to return data currently in the POGI file
+
+    Parameters
+    ----------
+    pause : multiprocessing.Lock
+        a lock shared between worker processes
+    exitRequest : multiprocessing.Queue
+        a queue that determines when the worker process stops
+    pipelineOut : multiprocessing.Queue
+        output pipeline for pogiData
+    """
+
+    print("Reading POGI data")
+
+    while True:
+        pause.acquire()
+        pause.release()
+
+        # get the pogi data
+        pogiData = read_pogi()
```

Comment:
Using the changes I mentioned above, `read_pogi()` should return a tuple where the first entry is a boolean representing if the pogi has been changed, and the second is the pogi data itself. Change this line to `changed, pogi_data = read_pogi()`.